lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -433,6 +438,17 @@ func (c *Client) startInputStreaming(stopCh chan struct{}) error {
 			return err
 		}
 
+		if m, ok := msg.(tdp.Ping); ok {
+			if !disableDesktopPing {
+				conn, err := net.Dial("tcp", c.cfg.Addr)
+				if err == nil {
+					conn.Close()
+				}
+			}
+			c.cfg.Conn.WriteMessage(m)
```

Comment:
What is the `WriteMessage` for? We receive a  `tdp.Ping` message and then we send the exact same message back?

Reply:
Yes, to avoid adding yet another tdp message like `Pong`

Reply:
Ok, I would add a brief comment:

```
// Upon receiving a ping message, we make a connection
// to the host and send the same message back to the proxy.
// The proxy will then compute the round trip time.
```

Also, should we have an error check on this `WriteMessage` call?

Reply:
@probakowski I'm still waiting a response on this one.

Reply:
I've added comment and added check for error